backend/api/Areas/Documents/Controllers/DocumentController.cs

```diff
@@ -0,0 +1,66 @@
+
+using MapsterMapper;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Pims.Api.Models.Concepts;
+using Pims.Api.Policies;
+using Pims.Api.Services;
+using Pims.Dal.Security;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Collections.Generic;
+
+namespace Pims.Api.Areas.Notes.Controllers
+{
+    /// <summary>
+    /// NoteController class, provides endpoints for interacting with notes.
+    /// </summary>
+    [Authorize]
+    [ApiController]
+    [ApiVersion("1.0")]
+    [Area("documents")]
+    [Route("v{version:apiVersion}/[area]")]
+    [Route("[area]")]
+    public class DocumentController : ControllerBase
```

Comment:
There is already a DocumentController at Controllers\DocumentController.cs

Reply:
Moved the endpoint in Controllers\DocumentController.cs